### PR TITLE
Write docs based on googledoc draft

### PR DESCRIPTION
**Pull request by mwilliams2014**

### CODE REVIEW

File: en/user/articles/100-web-experience-management/03-building-a-site/03-adding-pages-to-sites/04-widget-pages/04-converting-widget-pages-to-content-pages.markdown [hunk 1]
```diff
@@ -0,0 +1,119 @@
+# Converting Widget Pages to Content Pages
+
+In previous versions, While Content Pages provided many benefits, there were
```
Comment (ruben-pulido) [inline en/user/articles/100-web-experience-management/03-building-a-site/03-adding-pages-to-sites/04-widget-pages/04-converting-widget-pages-to-content-pages.markdown]: should "While" be "while"?

File: en/user/articles/100-web-experience-management/03-building-a-site/03-adding-pages-to-sites/04-widget-pages/04-converting-widget-pages-to-content-pages.markdown [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+**Note:** If you've already confirmed that a custom layout template can be 
+converted, You can disable the layout template conversion warning for the layout 
+template so you don't keep seeing it each time you convert a Widget Page that 
+uses the layout. Open the Product Menu and go to *Control Panel* &rarr; 
+*Configuration* &rarr; *System Settings*. Select *Pages* under Content and Data 
+and add the layout template ID to the list of "Verified Layout Template IDs" 
+under the System Scope.
+
+## Convert A Widget Page Directly to A Content Page
```
Comment (ruben-pulido) [inline en/user/articles/100-web-experience-management/03-building-a-site/03-adding-pages-to-sites/04-widget-pages/04-converting-widget-pages-to-content-pages.markdown]: Multiple widget pages can be converted at once using this method

**Comment by ruben-pulido** [comment]
Good job @mwilliams2014!
I've added 2 minor comments.

File: en/user/articles/100-web-experience-management/03-building-a-site/03-adding-pages-to-sites/04-widget-pages/04-converting-widget-pages-to-content-pages.markdown (hunk 1 above)
Comment (mwilliams2014) [reply en/user/articles/100-web-experience-management/03-building-a-site/03-adding-pages-to-sites/04-widget-pages/04-converting-widget-pages-to-content-pages.markdown]: Yep

**Comment by mwilliams2014** [comment]
Thanks for the feedback @ruben-pulido ! I will make the minor changes you suggested and merge this into the Creating Pages docs on Liferay Learn.

**Pull request closed without merging**